features/profile/components/organisms/my-page.tsx

```diff
@@ -3,8 +3,10 @@
 import { useState } from 'react';
 
 import { Button } from '@/components/atoms';
+import { StatisticsIcon } from '@/components/atoms';
 import BadgeIcon from '@/components/atoms/icons/badge-icon';
-import StatisticsIcon from '@/components/atoms/icons/statistics-icon';
+import { UserImageIcon } from '@/components/atoms/icons/user-image-icon';
```

Comment:
```suggestion
import { UserImageIcon } from '@/components/atoms/icons';
```

icons 폴더에 user-image-icon도 추가해주시면 좋을 것 같아요!